pkg/client/clients.go

```diff
@@ -69,19 +71,33 @@ func (c *Clients) GetConf() *conf.SchedulerConf {
 	return c.conf
 }
 
-func (c *Clients) WaitForSync(interval time.Duration, timeout time.Duration) error {
-	return utils.WaitForCondition(func() bool {
-		// cache is re-sync'd when all informers are sync'd
-		return c.NodeInformer.Informer().HasSynced() &&
+func (c *Clients) WaitForSync() {
+	timeNow := time.Now()
+	counter := 0
+	for {
+		if c.NodeInformer.Informer().HasSynced() &&
 			c.PodInformer.Informer().HasSynced() &&
 			c.PVCInformer.Informer().HasSynced() &&
 			c.PVInformer.Informer().HasSynced() &&
 			c.StorageInformer.Informer().HasSynced() &&
 			c.ConfigMapInformer.Informer().HasSynced() &&
 			c.NamespaceInformer.Informer().HasSynced() &&
 			c.PriorityClassInformer.Informer().HasSynced() &&
-			(c.AppInformer == nil || c.AppInformer.Informer().HasSynced())
-	}, interval, timeout)
+			(c.AppInformer == nil || c.AppInformer.Informer().HasSynced()) {
+			return
+		}
+		time.Sleep(time.Second)
+		counter++
+		if counter%10 == 0 {
+			if counter > 300 {
+				log.Logger().Warn("Still waiting for informers to sync, delay seems longer than usual...",
```

Comment:
I know I mentioned earlier that we could switch to a warning after a certain period of time, but after some more thought, I'd rather keep this consistent with the same message and INFO status. "Longer than usual" is both potentially incorrect and very opinionated. What's normal for one cluster may not be for another. The simple repeating of the INFO message every 10s should be sufficient. This can be monitored externally by checking the elapsed time field. To help with automated tools, we could also express it as a time in seconds rather than a Duration (i.e. `{"secondsElapsed": 300}`  instead of `{"timeElapsed": "5m0s"}`.